wsdl_rs/b_2/src/lib.rs

```diff
@@ -131,15 +137,45 @@ pub struct NotificationMessageHolderType {
 
 impl Validate for NotificationMessageHolderType {}
 
+#[derive(Default, PartialEq, Debug, YaSerialize, YaDeserialize)]
+#[yaserde(prefix = "tt", namespace = "tt: http://www.onvif.org/ver10/schema")]
+
+pub struct SimpleItemType {
+    // Item name.
+    #[yaserde(attribute, rename = "Name")]
+    pub name: String,
+
+    // Item value. The type is defined in the corresponding description.
+    #[yaserde(attribute, rename = "Value")]
+    pub value: String,
+}
+
 pub mod notification_message_holder_type {
     use super::*;
 
+    #[derive(Default, PartialEq, Debug, YaSerialize, YaDeserialize)]
+    #[yaserde(prefix = "tt", namespace = "tt: http://www.onvif.org/ver10/schema")]
+    pub struct DataType {
+        #[yaserde(prefix = "tt", rename = "SimpleItem")]
+        pub simple_item: SimpleItemType,
+    }
+
+    #[derive(Default, PartialEq, Debug, YaSerialize, YaDeserialize)]
+    #[yaserde(prefix = "tt", namespace = "tt: http://www.onvif.org/ver10/schema")]
+    pub struct MessageTypeInner {
+        #[yaserde(prefix = "tt", rename = "Data")]
+        pub data: DataType,
+    }
+
```

Comment:
Do you have an example XML from the camera? If yes, can you also add a test [here](https://github.com/lumeohq/onvif-rs/blob/a49b43d4713ee5ff5ffb7bab9105d6793c6c1085/schema/src/tests.rs)?